tunslip6.c

```diff
@@ -363,6 +375,99 @@ serial_to_tun(FILE *inslip, int outfd)
 unsigned char slip_buf[2000];
 int slip_end, slip_begin;
 
+int
+get_slipfd()
+{
+  int fd = -1;
+
+  ssystem("sleep %d", 10);
```

Comment:
This is really lame and I don't like it. Unfortunately we can't just sleep(3) because sigalarm is in use. I was thinking maybe fork() and then sleep() then wait on sleepy child? Input on this would be nice.